plugins/flutter_aepedge/test/flutter_aepedge_test.dart

```diff
@@ -177,4 +177,106 @@ void main() {
       expect(actualEventHandleResponse[1].type, expectedResponse[1].type);
     });
   });
+
+  group('setLocationHint', () {
+    final String testLocationHint = "irl1";
+    final List<MethodCall> log = <MethodCall>[];
+
+    setUp(() {
+      channel.setMockMethodCallHandler((MethodCall methodCall) async {
+        log.add(methodCall);
+        return null;
+      });
+    });
+
+    test('invokes correct method', () async {
+      await Edge.setLocationHint(testLocationHint);
+
+      expect(log, <Matcher>[
+        isMethodCall(
+          'setLocationHint',
+          arguments: testLocationHint,
+        ),
+      ]);
+    });
+  });
+
+  group('setLocationHint to null', () {
+    final String? testLocationHint = null;
+    final List<MethodCall> log = <MethodCall>[];
+
+    setUp(() {
+      channel.setMockMethodCallHandler((MethodCall methodCall) async {
+        log.add(methodCall);
+        return null;
+      });
+    });
+
+    test('invokes correct method', () async {
```

Comment:
Instead of defining `group('setLocationHint to null', `, can you just move this test to previous group?